nginx/templates/blog.cslant.com.template

```diff
@@ -1,10 +1,16 @@
 server {
-    listen ${NGINX_HOST_HTTP_PORT};
+    listen 80;
+    listen [::]:80;
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
```

Comment:
@pxthinh please add these 2 lines inside the custom ssl files. 
u are not allowed to add anything here because if you don't install it using SSL, it will cause an error